test/settings.test.ts

```diff
@@ -92,4 +92,18 @@ describe('StakingPool.settings', () => {
         .withArgs(testEnv.stakingPool.address, deployer.address, RAY);
     });
   });
+
+  context('reset the pool', async () => {
```

Comment:
We also need to check the rewardPerSecond, startTimestamp, endTimestamp are changed as we want.